packages/app/public/locales/en/common.json

```diff
@@ -25,7 +25,8 @@
   "actions": {
     "refresh": "Refresh data",
     "delete": "Delete",
-    "cancel": "Cancel"
+    "cancel": "Cancel",
+    "backToHome": "back to home"
```

Comment:
Starting with a capital, right? -> "Back to home" 🅱️

Reply:
Righty right!